tests/test_blocks.py

```diff
@@ -48,8 +49,8 @@ def test_columnar_result(self):
 
     def test_columnar_block_extend(self):
         with self.create_table('a Int32'):
-            self.client.execute('INSERT INTO test (a) VALUES', [(1, )])
```

Comment:
Let's keep PR clean and additional code untouched. These and the same changes below are unnecessary.

Reply:
I'm sorry, maybe i accidentally modify a space.